src/CabanaPD_Comm.hpp

```diff
@@ -251,6 +252,7 @@ class Comm<ParticleType, PMB>
     Comm( ParticleType& particles, int max_export_guess = 100 )
         : max_export( max_export_guess )
     {
+        _init_timer.start();
```

Comment:
@streeve In previous functions we had:
Timer _timer;
_timer.start();
_timer.stop();

Here we do not have "Timer _timer;"?

Also, why do we use "_init_timer" instead?

Reply:
For this class (not function) it's on lines 318-319